packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -77,6 +80,9 @@ export class Runner {
         try {
             await this.openPage(scanMetadata.url);
             axeScanResults = await this.scan(pageScanResult, this.page);
+
+            await this.deepScan(scanMetadata, pageScanResult);
```

Comment:
We are not ready to use crawler from runner. The crawler form cli does crawl **and** scan. What we need here is crawl only. The arch can look like crawler crawl and scan using runner impl similar to cli impl when scan processor is invoked from crawler.

Reply:
The crawler that's registered to the web-api-scan-runner container uses SimpleCrawlerEngine, which shares an interface with the cli CrawlerEngine, but only collects links from one web page without any scanning.

Reply:
Still this is not expected pattern. The `SimpleCrawlerEngine` crawler will create browser instance, make call to the web page, render the page, and crawl the page. All this duplicate runner except getting page links. We need to remove scanner inclusion from runner for now.